README.md

```diff
@@ -312,6 +313,23 @@ npm run release -- --no-verify
 commit-and-tag-version --no-verify
 ```
 
+### Custom Config Path
+
+Specify a custom path to the configuration file using the `--config` option
+
+```sh
+commit-and-tag-version --config ./path/to/.versionrc.js
+
+# or using alias
+commit-and-tag-version -c ./path/to/.versionrc.js
+```
+
+Supports all of the existing config extensions
```

Comment:
```suggestion
All config file formats can be used:
```
^ Or similar - `"the existing"` seemed a bit out of place here, as that's what you'd say in the PR description, rather than in the docs